packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/adddependency.md

```diff
@@ -4,8 +4,10 @@ The Okta Identity Engine SDK is available as a Swift package. Add it to your pro
 1. Choose the approriate dependency (usually "Up to Next Major Version").
 1. Click **Add Package**.
 
-Import the package into Swift files to access the SDK:
+The SDK includes a dependency on the Okta `AuthFoundation` library.
+Import the packages into Swift files to access the SDK:
```

Comment:
Was this meant to be on a separate line or following on from "AuthFoundation library"? At the moment it follows on.

Reply:
Nice catch. Fixed in next commit